### ISSUE TITLE
Faculty B# Submission Not Working

### ISSUE THREAD

**Issue by hmanalai** [BCStudentSoftwareDevTeam/urcpp#160]
**[Original report](https://bitbucket.org/laborstudents/urcpp-flask/issue/160) by Hailey Barnett (Bitbucket: [barnetth](https://bitbucket.org/barnetth), GitHub: [barnetth](https://github.com/barnetth)).**

----------------------------------------

When prompted to add Faculty’s B# the following error appears…

![](https://bitbucket.org/repo/e47RB5/images/4019173550-Capture.PNG)
This occurs even if the correct B# according to the database is entered.

Basically, /bnumbers doesn’t work

**Comment by hmanalai** [comment]
**Original comment by Katelyn Adams (Bitbucket: [paesh1997](https://bitbucket.org/paesh1997), GitHub: [paesh1997](https://github.com/paesh1997)).**

----------------------------------------

Also, it doesnt take the right format of B#s :\(

**Comment by hmanalai** [comment]
**Original comment by Hailey Barnett (Bitbucket: [barnetth](https://bitbucket.org/barnetth), GitHub: [barnetth](https://github.com/barnetth)).**

----------------------------------------

moved to issue #170
